Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -130,3 +171,31 @@ extension Schema {
         query.filters.append(.group([isNull, isFuture], .or))
     }
 }
+
+
+public protocol TimestampFormatter {
+    func string(from date: Date) -> String
+    func date(from string: String) -> Date?
+}
+
+extension DateFormatter: TimestampFormatter { }
+extension ISO8601DateFormatter: TimestampFormatter { }
+
+public enum TimestampFormat {
```

Comment:
Could we use an enum-like struct here instead to keep the cases internal? That would allow us to refactor this in the future if needed.

Reply:
One potential refactor we may wanna do is caching thread-specific formatters for example.